mycore-restapi/src/main/java/org/mycore/restapi/MCRRestFeature.java

```diff
@@ -86,7 +86,11 @@ protected void registerTransactionFilter(FeatureContext context) {
 
     @Override
     protected void registerAccessFilter(FeatureContext context, Class<?> resourceClass, Method resourceMethod) {
-        context.register(MCRRestAuthorizationFilter.class);
+        if (resourceClass.getPackageName().contains(".v1")) {
+            context.register(org.mycore.restapi.v1.MCRRestAuthorizationFilter.class);
+        } else {
+            context.register(org.mycore.restapi.v2.MCRRestAuthorizationFilter.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.mycore.restapi.v2.MCRRestAuthorizationFilter' due to existing same package import 'org.mycore.restapi.*'](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5532334)